package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amazon-chime-sdk-js",
-  "version": "1.1.3",
+  "version": "1.1.4",
```

Comment:
This seems wrong. Why are we bumping Sdk version?

Reply:
@asnyatkov the SDK bumps patch version with every commit now as part of `npm run build:release`.